docs/inventory-plugins.rst

```diff
@@ -1,229 +1,117 @@
 .. _ug_inventories_plugins:
 
 Inventory Plugins
-===================
-
-.. index::
-   pair: inventories; plugins
+=================
```

Comment:
separate?

Reply:
:man_shrugging: it shows up here, so it's new.

Reply:
You mean this section is new? It's not, it's been there: https://awx-plugins-core.readthedocs.io/en/latest/inventory-plugins/.  I thought you were commenting on the title length mismatch thing.

Reply:
You changed this line in the branch, which is why it shows up in the diff.